code/modules/paperwork/photocopier.dm

```diff
@@ -79,8 +79,7 @@
 		updateUsrDialog()
 	else if(href_list["remove"])
 		if(copyitem)
-			copyitem.loc = usr.loc
-			usr.put_in_hands(copyitem)
+			copyitem.forceMove(get_turf(src))
```

Comment:
This *does* lose the convenience of not having to click the paper when you're done. Perhaps a new proc could be made - `/atom/movable/proc/give_item(obj/item/thing, mob/living/target)` - that would either put the item in the person's hands if they were within a single-tile radius, or drop it on the atom's turf otherwise.